sql/driver/driver_test.go

```diff
@@ -297,73 +296,122 @@ CREATE TABLE t.users (
 	}
 }
 
-func TestInsertSelect(t *testing.T) {
+func TestInsertSelectDelete(t *testing.T) {
 	defer leaktest.AfterTest(t)
 	s, db := setup(t)
 	defer cleanup(s, db)
 
-	schema := `
-CREATE TABLE t.kv (
+	testcases := []struct {
+		db     string
+		schema string
+	}{
+		{
+			"t1",
+			`CREATE TABLE %s.kv (
   k CHAR PRIMARY KEY,
   v CHAR
-)`
+)`,
+		},
+		{
+			"t2",
+			`CREATE TABLE %s.kv (
+	k CHAR,
+	v CHAR,
+	PRIMARY KEY (k, v)
+)`,
+		},
+	}
+
+	for _, testcase := range testcases {
+		if _, err := db.Exec(fmt.Sprintf(`CREATE DATABASE ` + testcase.db)); err != nil {
```

Comment:
You could drop the table and database at the end if you wanted to reuse the database name. Fine the way it is, too. I think some more sophisticated testing infrastructure is going to be needed soon.